person.rb

```diff
@@ -5,8 +5,8 @@ def initalize(_age, name = 'Unknown', parent_permission: true)
     @age = int
     @parent_permission = parent_permission
   end
-  attr_reader :id
   attr_accessor :name, :age
+  attr_reader :id
 
   def can_use_services?
     if of_age || @parent_permission
```

Comment:
- [ ] How about using the descriptive name for the method like `is_of_age` instead of `of_age`.

Reply:
Fixed.